tests/tests/test-precompile/test-precompile-xcm-transactor.ts

```diff
@@ -361,13 +359,15 @@ describeDevMoonbeamAllEthTxTypes("Precompiles - xcm transactor", (context) => {
       [dest, asset, weight, transact_call]
     );
 
-    await context.createBlock(
-      createTransaction(context, {
-        ...ALITH_TRANSACTION_TEMPLATE,
-        to: PRECOMPILE_XCM_TRANSACTOR_ADDRESS_V1,
-        data,
-      })
-    );
+    const rawTxn = await createTransaction(context, {
+      ...ALITH_TRANSACTION_TEMPLATE,
+      to: PRECOMPILE_XCM_TRANSACTOR_ADDRESS_V1,
+      data,
+    });
+
+    console.log(rawTxn);
```

Comment:
Is this needed ?

Reply:
This was mistake but the whole file is getting archived now anyway.